src/main/java/io/vertx/redis/client/impl/cache/LRUClientSideCache.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ * <p>
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ * <p>
+ * The Eclipse Public License is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * <p>
+ * The Apache License v2.0 is available at
+ * http://www.opensource.org/licenses/apache2.0.php
+ * <p>
+ * You may elect to redistribute this code under either of these licenses.
+ */
+package io.vertx.redis.client.impl.cache;
+
+import io.vertx.redis.client.CachingRedisOptions;
+import io.vertx.redis.client.ClientSideCache;
+import io.vertx.redis.client.Response;
+
+public class LRUClientSideCache extends AbstractClientSideCache implements ClientSideCache {
+
+  public LRUClientSideCache(CachingRedisOptions options) {
+    super(options);
+  }
+
+  @Override
+  protected long valueAfterGet(CacheKey key, long currentValue) {
+    return System.nanoTime();
+  }
+
+  @Override
+  protected long initialValueOnSet(CacheKey key, Response value) {
+    return System.nanoTime();
```

Comment:
Should these be `System.currentTimeMillis()`, like the rest of the cache implementation code?